ENG-2002 - Send set password phone docs (#3705)
FusionAuth/fusionauth-site: astro/src/content/docs/apis/_tenant-request-body.mdx

    * `Gated` - verification is required before a user can complete login. The use of this value will require a paid plan.
  </APIField>

  <APIField name="tenant.phoneConfiguration.setPasswordTemplateId" type="UUID" optional since="1.99.9">

>     The Id of the Message Template used when a user had their account created for them and they must set their password manually and they are sent an SMS message to set their password.

Reviewer: ```suggestion
    The Id of the SMS Message Template used when a user must set their password manually after their account was created for them (by an admin , for example).
```

Author: sorry for the run on sentence
